FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -19,13 +19,33 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * http://www.FreeRTOS.org
- * http://aws.amazon.com/freertos
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
  *
+ * 1 tab == 4 spaces!
+ */
+
+/*
+ * Demo for showing how to use the Device Defender library's API. This version
+ * of the Device Defender API provides macros and helper functions for
+ * assembling MQTT topics strings, and for determining whether an incoming MQTT
+ * message is related to device defender. The Device Defender library does not
+ * depend on a MQTT library, therefore the code for MQTT connections is placed
+ * in another file (mqtt_demo_helpers.c) to make it easy to read the code using
+ * the Device Defender library.
+ *
```

Comment:
can you also mention that this demo uses coreMQTT library and if needed it can be replaced with any other MQTT library?